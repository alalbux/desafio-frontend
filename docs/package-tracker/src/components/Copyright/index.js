import React from 'react';
import Typography from '@mui/material/Typography';
import Link from '@/components/Link';

const COMPANY_NAME = 'Loggi';
const COMPANY_SITE = 'https://www.loggi.com/';

const Copyright = ({ props }) => {
  return (
    <Typography
      variant="body2"
      color="text.secondary"
      align="center"
      {...props}
    >
      {'Copyright Â© '}
      <Link color="inherit" href={COMPANY_SITE}>
        {COMPANY_NAME}
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
};

export default Copyright;
