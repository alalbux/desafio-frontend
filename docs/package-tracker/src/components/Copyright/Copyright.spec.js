import React from 'react';
import { render } from '@testing-library/react';
import Copyright from '@/components/Copyright';

describe('Copyright', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Copyright />);
    expect(container).toMatchSnapshot();
  });
});
