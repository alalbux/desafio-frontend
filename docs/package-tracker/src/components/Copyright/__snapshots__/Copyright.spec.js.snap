// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright matches the snapshot 1`] = `
<div>
  <p
    class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
  >
    Copyright Â© 
    <a
      class="MuiTypography-root MuiTypography-button MuiLink-root MuiLink-underlineNone css-17f0go1-MuiTypography-root-MuiLink-root"
      data-testid="link"
    />
     
    2023
    .
  </p>
</div>
`;
